Dockerfile

```diff
@@ -0,0 +1,24 @@
+############################
+# STEP 1 build executable binary
+############################
+FROM golang:alpine AS builder
+# Install git.
+# Git is required for fetching the dependencies.
+RUN apk update && apk add --no-cache ca-certificates git
+WORKDIR $GOPATH/src/github.com/mdelapenya/junit2otlp
+
+COPY . .
+
+# Build the binary.
+RUN GOOS=linux GOARCH=386 go build -ldflags="-w -s" -o /go/bin/junit2otlp
+############################
+# STEP 2 build a small image
+############################
+FROM scratch
+# Copy default certificates
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
What's the reason?

Reply:
It's important when the binary accesses resources under SSL/TLS. I think this could be the case for the tool when sending the OTEL traces/spans to a back-end. You can find more info [here](https://morioh.com/p/51d5ff070541#:~:text=First%2C%20let%E2%80%99s%20copy%20the%20ca-certificates%2C%20this%20is%20important%20if%20you%20will%20be%20using%20SSL/TLS%20to%20access%20your%20application.%20Otherwise%20you%20can%20skip%20this%20one%20if%20your%20app%20is%20not%20using%20SSL/TLS.)

Reply:
OTOH, we could remove it from the build until an hypothetical bug surfaces

Reply:
It makes sense to keep the current implementation. Thanks for the clarification